### PR TITLE
147

### PR DESCRIPTION
**Pull request by mbellew**
Fix for https://github.com/projectM-visualizer/projectm/issues/147
Seems to resolve the crashes.

I just noticed that this included a lot of whitespace changes.  Sorry about that.  The main change is in the header file anyway, plus update to renderFrame() and evaluateSecondPreset()

### CODE REVIEW

File: src/libprojectM/projectM.cpp [hunk 1]
```diff
@@ -70,29 +71,22 @@ projectM::~projectM()
 {
     void *status;
 
-    #ifdef USE_THREADS
+#ifdef USE_THREADS
     std::cout << "[projectM] thread ";
     printf("c");
```
Comment (revmischa) [inline src/libprojectM/projectM.cpp]: whats with the printfs

Comment (revmischa) [reply src/libprojectM/projectM.cpp]: Remove imo

Comment (revmischa) [reply src/libprojectM/projectM.cpp]: I mean are they printed? Is this code run?

Comment (mbellew) [reply src/libprojectM/projectM.cpp]: There is uncommented debug code in QPulseAudioThread.cpp and TextureManager.cpp as well.  Might be nice to have a runtime DEBUG flag as well as a compile time flag.  When I have a few minutes I could try to do a pass.

**Pull request merged**